installers/rpm-code/codes.sh

```diff
@@ -22,17 +22,26 @@ codes_cmake() {
     cmake -D CMAKE_RULE_MESSAGES:BOOL=OFF -D CMAKE_BUILD_TYPE:STRING="$t" "$@" ..
 }
 
+codes_cmake2() {
+    declare t=Release
+    if [[ ${CODES_DEBUG_FLAG:-} ]]; then
+        t=Debug
+    fi
+    cmake -S . -B build -D CMAKE_RULE_MESSAGES:BOOL=OFF -D CMAKE_BUILD_TYPE:STRING="$t" "$@"
```

Comment:
nice find

Reply:
The distinction with `codes_cmake2` is that it doesn't `cd build`, which is why I made it a separate function.